op/reboot.go

```diff
@@ -1,15 +1,15 @@
 package op
```

Comment:
I think this file is a little large.
Can you create a directory like `etcd` and split the file?

Reply:
If the problem is that this file is large, why not simply divide the file?

Reply:
Yes, I think so.